lib/src/nanotdf/Client.ts

```diff
@@ -128,8 +128,23 @@ export default class Client {
     ephemeralKeyPair?: CryptoKeyPair,
     dpopEnabled = false
   ) {
+    const enwrapAuthProvider = (a: AuthProvider): AuthProvider => {
```

Comment:
enwrap - is this a crypto term?

Reply:
I guess I should have called it proxied or delegated/delegating. I must just have wrapping on the mind

🎁